back/app/policies/phase_policy.rb

```diff
@@ -27,6 +27,10 @@ def survey_results?
     active_moderator?
   end
 
+  def sentiment_by_quarter?
+    active_moderator?
+  end
```

Comment:
Does this mean active moderator only for the Community Monitor project?

What if an admin wanted to create a global report + add in the Health Score and then share the link with other moderators on the platform (who are not moderating the Comm. Monitor project specifically)? Will that widget/data still show up for them? I'm not quite sure how this BE code works :)